apps/documentation/src/pages/_app.tsx

```diff
@@ -4,9 +4,5 @@ import '@voiceflow/chat/dist/style.css';
 import type { AppProps } from 'next/app';
 
 export default function App({ Component, pageProps }: AppProps) {
-  return (
-    <div>
-      <Component {...pageProps} />
-    </div>
-  );
+  return <div>{typeof window !== 'undefined' && <Component {...pageProps} />}</div>;
```

Comment:
Not sure we need this..?

Reply:
I think if we're going to have multiple places like this we should create a wrapper component for client-side-only rendering, it will be clearer why we're using them